### PR TITLE
Add xenondb

### PR DESCRIPTION
**Pull request by andyli029**

### CODE REVIEW

**Comment by andyli029** [comment]
@zheng1 
Please review the pr.
Thanks!

File: src/test/xenondb/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+
```
Comment (zheng1) [inline src/test/xenondb/README.md:1]: Just upload the files under the charts folder, the other files are useless.
You can reference other directory structure under src/test

File: src/test/xenondb/charts/README.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
+# Introduction
```
Comment (zheng1) [inline src/test/xenondb/charts/README.md:1]: This file should update to the latest version

File: src/test/xenondb/charts/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "5.7.33"
```
Comment (zheng1) [inline src/test/xenondb/charts/Chart.yaml:25]:
```
helm lint .
==> Linting .
[INFO] Chart.yaml: icon is recommended

1 chart(s) linted, 0 chart(s) failed
```

Comment (zheng1) [reply src/test/xenondb/charts/Chart.yaml:25]: Here are some of the more recommended fields
```
kubeVersion
sources
keywords
maintainers
```

File: src/test/xenondb/README.md (hunk 1 above)
Comment (andyli029) [reply src/test/xenondb/README.md:1]: got it.

**Comment by andyli029** [comment]
@zheng1
Please review the pr.

**Comment by andyli029** [comment]
@zheng1
Please review the pr.

File: src/test/xenondb/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
+apiVersion: v2
+name: xenondb
+description: Fast, reliable, scalable, and easy to use open-source relational database system.
```
Comment (zheng1) [inline src/test/xenondb/Chart.yaml:3]: `Open source, high availability cluster, based on MySQL`

File: src/test/xenondb/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+- email: jasongao@yunify.com
+  name: jason
+- email: runkecheng@yunify.com
+  name: rinker
+- email: molliezhang@yunify.com
+  name: molliezhang
+- email: dingyuan@yunify.com
+  name: dingyuan
+
+icon: https://github.com/radondb/xenondb/blob/main/docs/images/XenonDBLogo-F.svg
```
Comment (zheng1) [inline src/test/xenondb/Chart.yaml:52]: The icon is preferably the same length and width

**Pull request merged**